src/data/markdown/docs/03 cloud/05 Project and Team Management/04 SAML SSO.md

```diff
@@ -66,3 +70,17 @@ As you can see below, the default project is already selected:
 3. From the drop-down you can choose the default project(s) newly provisioned users will be assigned, then click on update button:
 ![SAML SSO update default projects](images/04-SAML-SSO/saml-sso-update-default-project.png)
 4. Now the users assigned to the IdP application can sign in to k6 Cloud and they'll be automatically added to the specified project(s) in step 3 with __project read/write__ permissions.
+
+## Fine-grained users to projects assignment using teams
+
+For automatic assignment of users to the projects on k6 Cloud, you can make use of the teams feature. With teams you can map groups on [Okta](/cloud/project-and-team-management/saml-sso/okta) or [Azure Active Directory](/cloud/project-and-team-management/saml-sso/azure-ad) with your desired projects on k6 Cloud.
```

Comment:
Maybe make this an active sentence instead, like 
> "The _Teams_ feature allows you to automatically assign all members of your internal teams (aka [Okta] and [Azure] groups) to one or more [k6 cloud projects]."

Reply:
Addressed.